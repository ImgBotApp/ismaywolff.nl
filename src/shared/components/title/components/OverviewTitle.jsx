import styled from 'styled-components'

const OverviewTitle = styled.h2`
  font-size: 0.75rem;
  margin-bottom: 0.5rem;
  margin-top: 0.5rem;
`

export default OverviewTitle
