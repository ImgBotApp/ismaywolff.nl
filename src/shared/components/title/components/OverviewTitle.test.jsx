/**
 * @jest-environment node
 */

import 'jest-styled-components'
import React from 'react'
import { shallow } from 'enzyme'
import OverviewTitle from './OverviewTitle'

describe('<OverviewTitle />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<OverviewTitle>Title</OverviewTitle>)
    expect(wrapper).toMatchStyledComponentsSnapshot()
  })
})
