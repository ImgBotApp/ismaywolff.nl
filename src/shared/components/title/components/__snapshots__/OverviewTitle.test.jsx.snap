// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OverviewTitle /> renders correctly 1`] = `
.bSmZHz {
  font-size: 0.75rem;
  margin-bottom: 0.5rem;
  margin-top: 0.5rem;
}

<h2
  className="sc-bdVaJa bSmZHz"
>
  Title
</h2>
`;
